index.js

```diff
@@ -1,88 +1,117 @@
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
+
+function drinkWater(currentThirstLevel) {
+  console.log("Man I sure am thirsty");
+  currentThirstLevel -= 1;
+  console.log("Ahh that hits the spot");
+  return currentThirstLevel
+}
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+console.log(drinkWater(12))
+// => "Man I sure am thirsty"
+// => "Ahh that hits the spot"
+// => 11
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
-
-// console.log(sameSameButDifferent(5, 5)) 
-// // => "same same"
+console.log("%cQuestion 4", "color: red") 
+
+function sameSameButDifferent(num, maybeNum) {
+  if (num === maybeNum) {
+    return "same same";
+  } else if (num == maybeNum) {
+    return "same same (but different)";
+  } else {
+    return "different";
+  }
+}
+console.log(sameSameButDifferent(5, 5)) 
+// => "same same"
 
-// console.log(sameSameButDifferent(123, "123")) 
-// // => "same same (but different)"
+console.log(sameSameButDifferent(123, "123")) 
+// => "same same (but different)"
 
-// console.log(sameSameButDifferent(5, 7)) 
-// // => "different"
+console.log(sameSameButDifferent(5, 7)) 
+// => "different"
 
-// console.log(sameSameButDifferent(123, "122")) 
-// // => "different"
-// console.log("%c----------", "color: red") 
+console.log(sameSameButDifferent(123, "122")) 
+// => "different"
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 5 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 5", "color: red")
-// const student1 = { name: "Duane", grade: 88 }
-// updateGrade(student1, 92)
-// console.log(student1)
-// // => { name: "Duane", grade: 92 }
-// console.log("%c----------", "color: red")
+console.log("%cQuestion 5", "color: red")
+
+function updateGrade(student, number) {
+  student.grade = number;
+}
+const student1 = { name: "Duane", grade: 88 }
+updateGrade(student1, 92)
+console.log(student1)
+// => { name: "Duane", grade: 92 }
+console.log("%c----------", "color: red")
 
 
 
 // ***** Question 6 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 6", "color: red")
-
-// const users = [ 
-//   { 
-//     name: "Duane", phones: { cell: "555-123-4567", office: "555-456-7890" }
-//   },
-//   { 
-//     name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
-//   }
-// ]
-// printNameAndPhones(users)
-// // => "Duane"
+console.log("%cQuestion 6", "color: red")
+
+function printNameAndPhones(arr) {
+  arr.forEach(function callBack(value, i, arr) {
+    console.log(arr[i]["name"]);
+    console.log(`Cell: ${arr[i]["phones"]["cell"]}`);
+    console.log(`Office: ${arr[i]["phones"]["office"]}`);
+  })
+}
```

Comment:
Nice job making this work. We can simplify your code by using the built-in power of forEach. forEach is going to iterate over the array "users" below. We can use "user" to represent each instance in the array and then put that inside a callback function.
users.forEach(function callback(user) {}

Since callback only exists inside of printNameAndPhones and will never be called elsewhere, we can actually make "callback" into an anonymous function and remove its name
users.forEach(function(user) {}

Now we can rewrite the console.log 's without having to do arr[i].  You can use dot or bracket notation 

    console.log(user.name)
    console.log("Cell:", user['phones']['cell'])
    console.log("Office:", user.phones.office)


function printNameAndPhones(users) {
  users.forEach(function(user) {
    console.log(user.name)
    console.log("Cell:", user['phones']['cell'])
    console.log("Office:", user.phones.office)
  })
}

The most important thing is that you were able to make it work. What I provided would be a good refactor.